File: blog-dao/src/main/java/com/dao/TypeMapper.java
package com.dao;

import com.entity.Type;

public interface TypeMapper {
    int deleteByPrimaryKey(Integer tid);

    int insert(Type record);

    int insertSelective(Type record);

    Type selectByPrimaryKey(Integer tid);

    int updateByPrimaryKeySelective(Type record);

    int updateByPrimaryKey(Type record);

    //通过类别进行查询
    Type selectByType(String type);

}

File: blog-dao/src/main/java/com/dao/TypeMapper.kt
package com.dao

import com.entity.Type

interface TypeMapper {
    fun deleteByPrimaryKey(tid: Int?): Int

    fun insert(record: Type): Int

    fun insertSelective(record: Type): Int

    fun selectByPrimaryKey(tid: Int?): Type

    fun updateByPrimaryKeySelective(record: Type): Int

    fun updateByPrimaryKey(record: Type): Int


}